SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -26,6 +26,10 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialActorChannel);
 
+DECLARE_CYCLE_STAT(TEXT("SpatialActorChannel ~ ReplicateActor"), STAT_SpatialActorChannelReplicateActor, STATGROUP_SpatialNetDriver);
+DECLARE_CYCLE_STAT(TEXT("SpatialActorChannel ~ UpdateSpatialPosition"), STAT_SpatialActorChannelUpdateSpatialPosition, STATGROUP_SpatialNetDriver);
+DECLARE_CYCLE_STAT(TEXT("SpatialActorChannel ~ ReplicateSubobject"), STAT_SpatialActorChannelReplicateSubobject, STATGROUP_SpatialNetDriver);
```

Comment:
No need to do the `SpatialActorChannel ~` bit imo. It's obvious from `STAT` name if anyone wants to disambiguate.

Reply:
Just copied the style from Unreal where I found the code originally, however they are inconsistent in the code base anyway so don't mind making this change.